test/test_hermite.py

```diff
@@ -70,28 +69,110 @@ def test_hermite_tetrahedron():
     # Hermite basis functions taken from Symfem
     sym_basis = [
         (
```

Comment:
removing this open bracket would make the formatting here a little more consistent